apps/system/utils/Kconfig

```diff
@@ -11,7 +11,7 @@ config SYSTEM_CMDS
 		This configuration just registers enabled system commands in TASH, not enables
 		system commands. Because they can be used without TASH by function calling.
 		Let's enable system commands at each configuration.
-
+if !ARCH_ARMV7A
```

Comment:
We cannot have a check for V7A at the app level. It does not look neat. Let us look for an alternative approach if required.